libs/feature/record/src/lib/data-view-map/data-view-map.component.spec.ts

```diff
@@ -430,32 +430,6 @@ describe('DataViewMapComponent', () => {
       })
     })
 
-    describe('with a link using ESRI:REST protocol', () => {
-      beforeEach(fakeAsync(() => {
-        mdViewFacade.mapApiLinks$.next([])
-        mdViewFacade.geoDataLinks$.next([
-          {
-            protocol: 'ESRI:REST',
-            name: 'mes_hdf',
-            url: 'https://services8.arcgis.com/rxZzohbySMKHTNcy/arcgis/rest/services/mes_hdf/WFSServer/0',
```

Comment:
Maybe it could make sense to just change this test to a `FeatureServer` (surprised I don't see one yet)?

Reply:
Yes, this is indeed missing, thanks!